### PR TITLE
Redesign Canvas (Moveable) selection

### LINKED ISSUE #6200: Redesign: Canvas selection
## Feature Description

Update design for canvas selection based on [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1009%3A4323).

- [x] Update edge thickness and color
- [x] Update handle color, border and shape

| Current design | New design |
|-|-|
| Screen Shot 2021-02-04 at 22 16 09 | Screen Shot 2021-02-04 at 22 16 16 |

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## User-facing changes
Canvas selection has now been updated to [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1009%3A4323).

## Testing Instructions
See the issue [description](https://github.com/google/web-stories-wp/issues/6200) with the todo list and verify style matching based on Figma.

Before:
Screenshot 2021-02-08 at 11 33 09

After:
Screenshot 2021-02-08 at 11 32 35

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Add an element, e.g. a square shape.
2. Select the element
3. Verify the selection matching the style in Figma.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6200

### CODE REVIEW

File: assets/src/edit-story/theme.js [hunk 1]
```diff
@@ -81,6 +85,7 @@ export function useTheme(selector) {
 }
 
 const theme = {
+  designSystemTheme,
```
Comment (miina) [inline assets/src/edit-story/theme.js]: @barklund FYI I'm adding this into `theme` for being able to use the new colors from `design-system`, we can later replace it then. LMK if you had something else in mind.

Comment (barklund) [reply assets/src/edit-story/theme.js]: I think we should figure out "the right way" to use the new design system theme in #6209, so this is fine for now.

File: assets/src/edit-story/components/moveable/moveStyle.js [hunk 2]
```diff
@@ -22,10 +22,11 @@ import { createGlobalStyle } from 'styled-components';
...
 	.default-moveable .moveable-control,
 	.default-moveable .moveable-line.moveable-rotation-line .moveable-control {
-		background: ${({ theme }) => theme.colors.accent.primary} !important;
-		border: 2px solid #fff !important;
-		margin-left: -6px;
-		box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+		background: ${({ theme }) =>
+      theme.designSystemTheme.colors.bg.primary} !important;
+		border: 1px solid ${({ theme }) =>
+      theme.designSystemTheme.colors.blue[20]} !important;
```
Comment (barklund) [inline assets/src/edit-story/components/moveable/moveStyle.js]: Just [asked for clarification](https://xwp.slack.com/archives/C01ESBB8ZN1/p1612820685057600) about how to refer to these "non-standard" shades of the basic colors. I had the impression, that we only should use the named colors identified and not the full range.

Comment (barklund) [reply assets/src/edit-story/components/moveable/moveStyle.js]: @samitron7 confirms that this is an error - it should be blue-30, which is known as `border.focus` in the system:
Screen Shot 2021-02-08 at 17 03 56


We should never refer directly to the "raw" colors and if a design is using one, it's most likely a mistake.

@samitron7 will update the design to reflect this.

File: assets/src/edit-story/components/moveable/moveStyle.js [hunk 3]
```diff
@@ -42,38 +43,58 @@ export const GlobalStyle = createGlobalStyle`
...
 	.default-moveable .moveable-control.moveable-w {
-		border-radius: 8px;
+		border-radius: 4px;
+	}
+
+	.default-moveable .moveable-control.moveable-nw,
+	.default-moveable .moveable-control.moveable-ne,
+	.default-moveable .moveable-control.moveable-sw,
+	.default-moveable .moveable-control.moveable-se {
+	  border-radius: 0px;
```
Comment (barklund) [inline assets/src/edit-story/components/moveable/moveStyle.js]: This should be 2px, no?
Screen Shot 2021-02-08 at 17 13 26

**Review by barklund — COMMENTED** [review]
Looks good (a bit menacing even), just a few comments.

File: assets/src/edit-story/components/moveable/moveStyle.js (hunk 2 above)
Comment (barklund) [reply assets/src/edit-story/components/moveable/moveStyle.js]: We ended up deciding that we'll add a now border property named `border.selection` - it'll be the same color as focus, but more properly named (and could be changed independently), so please update to that once @BrittanyIRL merges.

**Review by barklund — APPROVED** [review]
Pending merge with #6209, this looks good.

**Pull request merged**